stylelint.config.js

```diff
@@ -27,9 +30,260 @@ module.exports = {
-      "declarations"
+      [
+				"dollar-variables",
+				"custom-properties",
+				"at-rules",
+				"declarations",
+				{
+					type: "at-rule",
+					name: "media",
+        },
+        "rules",
+			]
     ],
+    "order/properties-order": [
+			[
+				"content",
+				"position",
+				"top",
+				"right",
+				"bottom",
+				"left",
+				"z-index",
+				"display",
+				"vertical-align",
+				"flex",
+				"flex-grow",
+				"flex-shrink",
+				"flex-basis",
+				"flex-direction",
+				"flex-flow",
+				"flex-wrap",
+				"grid",
+				"grid-area",
+				"grid-template",
+				"grid-template-areas",
+				"grid-template-rows",
+				"grid-template-columns",
+				"grid-row",
+				"grid-row-start",
+				"grid-row-end",
+				"grid-column",
+				"grid-column-start",
+				"grid-column-end",
+				"grid-auto-rows",
+				"grid-auto-columns",
+				"grid-auto-flow",
+				"grid-gap",
+				"grid-row-gap",
+				"grid-column-gap",
+				"gap",
+				"row-gap",
+				"column-gap",
+				"align-content",
+				"align-items",
+				"align-self",
+				"justify-content",
+				"justify-items",
+				"justify-self",
+				"order",
+				"float",
+				"clear",
+				"object-fit",
+				"overflow",
+				"overflow-x",
+				"overflow-y",
+				"overflow-scrolling",
+				"clip",
+
+				//
+				"box-sizing",
+				"width",
+				"min-width",
+				"max-width",
+				"height",
+				"min-height",
+				"max-height",
+				"margin",
+				"margin-top",
+				"margin-right",
+				"margin-bottom",
+				"margin-left",
+				"padding",
+				"padding-top",
+				"padding-right",
+				"padding-bottom",
+				"padding-left",
+				"border",
+				"border-spacing",
+				"border-collapse",
+				"border-width",
+				"border-style",
+				"border-color",
+				"border-top",
+				"border-top-width",
+				"border-top-style",
+				"border-top-color",
+				"border-right",
+				"border-right-width",
+				"border-right-style",
+				"border-right-color",
+				"border-bottom",
+				"border-bottom-width",
+				"border-bottom-style",
+				"border-bottom-color",
+				"border-left",
+				"border-left-width",
+				"border-left-style",
+				"border-left-color",
+				"border-radius",
+				"border-top-left-radius",
+				"border-top-right-radius",
+				"border-bottom-right-radius",
+				"border-bottom-left-radius",
+				"border-image",
+				"border-image-source",
+				"border-image-slice",
+				"border-image-width",
+				"border-image-outset",
+				"border-image-repeat",
+				"border-top-image",
+				"border-right-image",
+				"border-bottom-image",
+				"border-left-image",
+				"border-corner-image",
+				"border-top-left-image",
+				"border-top-right-image",
+				"border-bottom-right-image",
+				"border-bottom-left-image",
+
+				//
+				"background",
+				"background-color",
+				"background-image",
+				"background-attachment",
+				"background-position",
+				"background-position-x",
+				"background-position-y",
+				"background-clip",
+				"background-origin",
+				"background-size",
+				"background-repeat",
+				"color",
```

Comment:
this should be with fonts IMO

Reply:
i'm okay with how it is now, but let me know if you do want a change.

Reply:
lets move it down